api/src/client/typescript/faces/impl/AjaxImpl.ts

```diff
@@ -208,7 +208,7 @@ export module Implementation {
 
         /* run through all script tags and try to find the one that includes faces.js */
         const foundStage = ExtDomQuery.searchJsfJsFor(/stage=([^&;]*)/).value as string;
-        return (foundStage in ProjectStages) ? foundStage : ProjectStages.Production; // MYFACES-4572: default is production
+        return (foundStage in ProjectStages) ? foundStage : null;
```

Comment:
this reverts a fix from @volosied

Reply:
@werpu why you dont work on a MF core fork? We cant expect that everyone clones 2 repos for a change in MF core

Reply:
Just pushed the fix! 
Thanks a lot, this would have been lost!